new-lamassu-admin/src/components/expandable-table/ExpTable.js

```diff
@@ -28,7 +28,7 @@ const ExpRow = ({ id, columns, details, sizes, expanded, className, expandRow, .
 
   const detailsRowClasses = {
     [classes.detailsRow]: true,
-    [classes.hideDetailsRow]: expanded
+    [classes.hideDetailsRow]: !expanded
   }
 
   return (
```

Comment:
Why are you using uuidv1 on all the components here? You assign key for the components only for the react internals, rule of thumb is using `idx`. If you generate the key every time it will be as if you're handing a new component every time.

Reply:
Also I don't even see uuid lib on package.json, so it wouldn't work on production.